pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx

```diff
@@ -25,15 +25,15 @@ export default function (props: GraphDashboardProps) {
     </LineGraph>,
 
     <LineGraph title="RPC Errors" sources={nodeSources}>
-      <Axis units={AxisUnits.Count} label="rpc error type">
+      <Axis units={AxisUnits.Count} label="errors">
```

Comment:
these `units` properties aren't needed, since the default prop is `AxisUnits.Count` anyway